ci-operator/config/openshift/multiarch-manager-operator/openshift-multiarch-manager-operator-main.yaml

```diff
@@ -10,6 +14,12 @@ images:
 - dockerfile_path: Dockerfile
   from: ubi
   to: multiarch-manager-operator
+operator:
+  bundles:
+  - as: multiarch-manager-operator-bundle
```

Comment:
i think there should be  a `dockerfile_path: bundle.Dockerfile` here so that it knows how to build the bundle  ?

Reply:
it should be the default, based on https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/

Reply:
It wasn't finally. We'd change that doc. thanks @Prashanth684